core/src/test/java/org/wildfly/extras/creaper/core/offline/OfflineServerVersionTest.java

```diff
@@ -296,6 +317,23 @@ public void discoverHostXml_wfly28() throws IOException {
         test(ServerVersion.VERSION_21_0_0, HOST_XML, WFLY28_ROOT, EAP7_LOGGING, EAP8_EE);
     }
 
+    @Test
+    public void discoverHostXml_wfly34_community() throws IOException {
+        test(ServerVersion.VERSION_20_0_0, HOST_XML, COMMUNITY, WFLY27_ROOT, EAP7_LOGGING, EAP8_EE);
```

Comment:
Just note: the version of root element does not correspond to the test target but we will be updating supported WF versions in next PR so might fix it there.

Reply:
+1 there is no guarantee that the schema version will even be updated on a major WildFly version so many WildFly versions will share the same schema version even if they have distinct model versions.